### PR TITLE
Add Super-Linter Github Actions configuration files

### PR DESCRIPTION
**Pull request by davidxbors**
Add configuration files needed in order to integrate Super-Linter to any Github repository.

Signed-off-by: David Bors <daviddvd267@gmail.com>

### CODE REVIEW

File: github/workflows/rules/md101.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+const { InlineTokenChildren } = require("./common/inlineTokenChildren");
+const { WordPattern } = require("./common/wordPattern");
+
+const keywords = [
+    new WordPattern("iExtractor-manager"),
```
Comment (teodutu) [inline github/workflows/rules/md101.js:5]: This is specific to Malus Security. It does no harm to us, but do we need it?

**Review by teodutu — COMMENTED** [review]
Is this supposed to be a guideline / template Super-Linter action? If not, then you should also add the changes you made to the action inside the OS repo: https://github.com/systems-cs-pub-ro/operating-systems-oer/pull/10. Otherwise, it should be fine.

File: github/workflows/rules/md101.js (hunk 1 above)
Comment (razvand) [reply github/workflows/rules/md101.js:5]: @davidxbors , remove it.

**Comment by davidxbors** [comment]
> @davidxbors , create this into a `templates/github/` folder. I will create a separate PR with the actual Super-Linter.
> 
> This is supposed to be a template setup to be used by others: copy-paste contents from `templates/github/` to `.github` in their repository.

Done.

File: github/workflows/rules/md101.js (hunk 1 above)
Comment (davidxbors) [reply github/workflows/rules/md101.js]: Yes, it's supposed to be a template. I removed `iExtractor-manager` and other Malus Security specific keywords.

**Review by teodutu — APPROVED** [review]
Looks legit.

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**